test/integration/tools/ros/test_rosenv.py

```diff
@@ -0,0 +1,68 @@
+import os
+import textwrap
+import platform
+
+import pytest
+
+from conan.test.utils.tools import TestClient
+
+
+@pytest.mark.skipif(platform.system() == "Windows", reason="Uses UNIX commands")
+def test_rosenv():
+    """
+    Test that the amentdeps generator generates conan_<name> folders and place CMake files
+    in the correct path
+    """
+    client = TestClient()
+    conanfile1 = textwrap.dedent('''
+        import os
+        from conan import ConanFile
+        from conan.tools.files import save
+        class Recipe(ConanFile):
+            name = "lib1"
+            version = "1.0"
+            settings = "os", "compiler", "build_type", "arch"
+
+            def package(self):
+                save(self, os.path.join(self.package_folder, "lib", "lib1"), "")
```

Comment:
Is it really needed?
It seems dependencies are completely superfluous in this test.